sites/public/pages/listings.tsx

```diff
@@ -257,7 +316,7 @@ const ListingsPage = () => {
             size={AppearanceSizeType.small}
             styleType={AppearanceStyleType.secondary}
             // "Submit" the form with no params to trigger a reset.
-            onClick={() => onSubmit({})}
+            onClick={() => setQueryString(1, null)}
```

Comment:
I think this should still be calling `onSubmit()` right?

Reply:
In this case it doesn't matter because the filter modal isn't open; I changed it back though. I had to pass null instead of an empty object because the `$comparison` property is required.